### PR TITLE
Adjusted per https://w3c.github.io/w3c.json.html

### PR DESCRIPTION
**Pull request by plehegar**
Tweak

### CODE REVIEW

File: w3c.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
  {
-    "group":      ["34314"]
+    "group":      34314
 ,   "contacts":   ["plehegar"]
-,   "shortName":  "ttml-webvtt-mapping"
```
Comment (nigelmegitt) [inline w3c.json:4]: Why remove the shortName?

**Review by nigelmegitt — APPROVED** [review]
Puzzled about the shortName, otherwise looks good to me.

File: w3c.json (hunk 1 above)
Comment (plehegar) [reply w3c.json:4]: we're not using it.

Comment (nigelmegitt) [reply w3c.json:4]: Thanks for the info @plehegar .

**Pull request merged**